GameData/RealismOverhaul/RemoteTech_Settings.cfg

```diff
@@ -1,9 +1,14 @@
-RemoteTechSettings:NEEDS[RealismOverhaul]
+//  ==================================================
+//  Custom RemoteTech settings for Realism Overhaul.
+
+//  *   Make "Root" the default communication rule set.
+//  *   Allow combining multiple antennae for range extension.
+//  *   Zero the active engine throttle on connection loss.
```

Comment:
Theoretically once you lose control of a craft you cannot do anything. So, if your engines are firing and you lose control then the throttle should remain as-is. I done goofed and wrote a bad comment there...